pulsar-client-cpp/tests/SchemaTest.cc

```diff
@@ -107,3 +107,60 @@ TEST(SchemaTest, testHasSchemaVersion) {
 
     client.close();
 }
+
+TEST(SchemaTest, testKeyValueSchema) {
+    // 0. test key value only support JSON or AVRO.
```

Comment:
> key value only support JSON or AVRO.

It's wrong. Run the following Java application:

```java
        final String topic = "my-schema-topic-" + System.currentTimeMillis();
        @Cleanup
        PulsarClient client = PulsarClient.builder().serviceUrl("pulsar://localhost:6650").build();

        BiConsumer<KeyValueEncodingType, KeyValue<String, String>> send = (type, keyValue) -> {
            try (Producer<KeyValue<String, String>> producer = client
                    .newProducer(Schema.KeyValue(Schema.STRING, Schema.STRING, type)).topic(topic).create()) {
                producer.send(keyValue);
            } catch (PulsarClientException e) {
                throw new RuntimeException(e);
            }
        };
        send.accept(KeyValueEncodingType.INLINE, new KeyValue<>("key-0", "value-0"));
        send.accept(KeyValueEncodingType.SEPARATED, new KeyValue<>("key-1", "value-1"));
```

Did you only see the `Schema#KeyValue` methods with `Class<?>` arguments? Maybe you can get more info from my email here: https://lists.apache.org/thread/4r3rmgjlxbgppfyglxyg6rfn6v42hvjf

Reply:
Oh, Yes, I see it.